lib/views/pages/home/content/map/map_screen.dart

```diff
@@ -52,7 +34,9 @@ class MapScreen extends ConsumerWidget {
         );
       },
       fallbackBuilder: (context, error) {
-        return fallback;
+        return ErrorRefreshPage(
+          onRefresh: () => ref.read(mapViewModelProvider.notifier).refresh(),
```

Comment:
Here I think that we could declare a final function above like you did for the rest of the refactor.

Reply:
I added a variable in the other screens because the function was reused multiple times, why do you think I need to add one here ?

Reply:
You're completely right, I didn't think about that!